Stormshield/stormshield_module/process_actions.py

```diff
@@ -0,0 +1,9 @@
+from stormshield_module.base import StormshieldAction
+
+base_url = "/agents/{id}/tasks"
+
+
+class TerminateProcessAction(StormshieldAction):
+    verb = "post"
+    endpoint = base_url + "/process-termination"
+    query_parameters: list[str] = ["processPath", "terminateProcessTree"]
```

Comment:
These fields must be passed in the request body; not as query parameters
```suggestion
    query_parameters: list[str] = []
```